toolkit/tools/imager/imager.go

```diff
@@ -511,3 +521,7 @@ func configureDiskBootloader(systemConfig configuration.SystemConfig, installChr
 
 	return
 }
+
+func isRootFs(config configuration.SystemConfig) bool {
```

Comment:
Please use a different name, so it doesn't overlap with the variable `isRootFs`. We wouldn't want to by accident end up using a function pointer somewhere - might be an unnecessary headache to figure out why the tools won't compiler/run correctly.